pretyping/structures.mli

```diff
@@ -0,0 +1,164 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *         Copyright INRIA, CNRS and contributors             *)
+(* <O___,, * (see version control and CREDITS file for authors & dates) *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
+(************************************************************************)
+
+
+(** A projection to a structure field *)
+module Projection : sig
```

Comment:
I don't like that this conflicts with Names.Projection, it confused me when looking at the `projections : Projection.t list` below.

Reply:
I've renamed it.